### ISSUE TITLE
Dead project?

### ISSUE THREAD

**Issue by dominikj111** [maniacbug/StandardCplusplus#32]
Last commit on 22 Sep 2013 and it is doesn't work on Platformio and Arduino UNO. I'm getting plenty errors when doing build. But works with the ArduinoSTL lib.

The errors:
```
...
.pio/libdeps/uno/StandardCplusplus/ostream:318:50: error: default argument for template parameter for class enclosing 'class std::basic_ostream<charT, traits>::sentry'
.pio/libdeps/uno/StandardCplusplus/istream:343:107: error: default argument for template parameter for class enclosing 'class std::basic_istream<charT, traits>::sentry'
.pio/libdeps/uno/StandardCplusplus/ostream:318:50: error: default argument for template parameter for class enclosing 'class std::basic_ostream<charT, traits>::sentry'
...
```

**Comment by chen-lee** [comment]
Just find the error location of the code and then delete '= char_traits<charT>', i tried this, it works.
![image](https://user-images.githubusercontent.com/31262436/147872671-0ead5485-6b01-408b-8b51-ba1d7eebfc6d.png)

**Comment by maniacbug** [comment]
Should be fixed now in commit 9d8277e
